### PR TITLE
369 Netlify toml

### LINKED ISSUE #369: Deploy Apollos 6.0.0 to Prod
## Acceptance Criteria

- [ ] Apollos 6.0.0 should be deployed to NewSpring Production environment.

## Context

We need to research and construct a solution for deployment of Apollos 6.0.0 on the web.

### PR DESCRIPTION
**Pull request by IsaacHardy**
First part for fixing #369.

* This updates the script to build web to run with different node environments
* It creates a netlify.toml file to run different commands based on a specific branch. 
* It updates the `create-env-files` script to let us use `.env` file instead of keeping that information on Netlify. This should allow us to set certain branches up to be "beta" and "alpha" with specific env variables for each all the while keeping this data closed from the public. 

## Creator

- [ ] Build relevant tests if any apply
- [ ] Ensure there are no new warnings
- [ ] Upload an animated GIF showcasing the solution (if it applies)
- [ ] Set a relevant reviewer

## Reviewer

- [x] Run `test` and make sure all tests pass
- [x] Review function and form on iOS
- [x] Review function and form on Android
- [x] Review function and form on Web
- [x] Review new test logic

### CODE REVIEW

File: .scripts/generate-env-files.js [hunk 1]
```diff
@@ -40,20 +41,14 @@ APP_HEIGHLINER_URL='${settings.prod.heighliner}'
...
-      if (err) console.log(err);
-      console.log("Local environment settings created.");
-    })
-    await FS.appendFile(Path.resolve(__dirname, '../.env.test'), testContent, (err) => {
-      if (err) console.log(err);
-      console.log("Test environment settings created.");
-    })
-    await FS.appendFile(Path.resolve(__dirname, '../.env.production'), productionContent, (err) => {
+    const envContent =
+      env === 'production' ? productionContent : env === 'test' ? testContent : localContent;
```
Comment (burkeshartsis) [inline .scripts/generate-env-files.js]: Is there any way to avoid this nested ternary 🤮

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+[build]
+# This is the directory that you are publishing from (relative to root of your repo)
+  publish = "build/"
+# default build command
+  command = "npm run build:alpha"
+
+# Production context: All deploys from the Production branch set in your site's deploy settings will inherit these settings.
+[context.production]
+  command = "npm run build"
```
Comment (burkeshartsis) [inline netlify.toml]: let's point all of these at alpha for the time being so nothing breaks.

File: .scripts/generate-env-files.js (hunk 1 above)
Comment (IsaacHardy) [reply .scripts/generate-env-files.js]: 😆 I could go the ole' fashion way, but didn't think it was that terrible with only three options.

**Comment by IsaacHardy** [comment]
@burkeshartsis Updated with new [context.production] command.

File: .scripts/generate-env-files.js (hunk 1 above)
Comment (burkeshartsis) [reply .scripts/generate-env-files.js]: 🙃

Comment (IsaacHardy) [reply .scripts/generate-env-files.js]: hahahahaha

**Pull request merged**